python_modules/dagster/dagster_tests/daemon_tests/test_queued_run_coordinator_daemon.py

```diff
@@ -479,7 +479,7 @@ def test_limits_per_unique_value(use_threads, workspace_context, pipeline_handle
 
         list(daemon.run_iteration(bounded_ctx))
 
-        assert get_run_ids(instance.run_launcher.queue()) == ["run-1", "run-3"]
+        assert set(get_run_ids(instance.run_launcher.queue())) == {"run-1", "run-3"}
```

Comment:
I think this is worth a comment. Seeing a "queue" and then adding sets around it to make order not matter is an eyebrow raiser. I imagine adding threads loosened the ordering guaranteed of this queue